src/OpenTelemetry.Extensions.Hosting/Implementation/EventSourceEventFormatting.cs

```diff
@@ -0,0 +1,93 @@
+// <copyright file="EventSourceEventFormatting.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics.Tracing;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+
+namespace OpenTelemetry.Internal
+{
+    internal static class EventSourceEventFormatting
```

Comment:
nit: `EventSourceEventFormatting` sounds more like a namespace, maybe `EventSourceEventFormatter` would be more standard? 🤷

Reply:
I agree with that. Changed.